tox.ini

```diff
@@ -0,0 +1,12 @@
+[tox]
+envlist = py37,pep8
+sitepackages=True
+
+[testenv]
+sitepackages=True
```

Comment:
Is this necessary? All the required packages listed in requirements.txt can be installed into a virtual environment.

Reply:
without this some things are left out, for instance dbus python module, I could chase down the missing dependencies (most likely gtk and friends) so we could remove this line.

Reply:
dbus python module is included in https://github.com/getting-things-gnome/gtg/commit/d9c4a3bb710bc693bd83baa0235afafe4e0220e9

Currently, I'm running gtg just from a venv created without `--system-site-packages`.

Reply:
Tox still runs fine for me without those lines

Reply:
Friendly ping. @federicomenaquintero do you want to try to add the dependencies? Or are they not needed? @tkdchen what do you think?